tool-box/Dockerfile

```diff
@@ -1,16 +1,32 @@
-FROM fedora:latest
+FROM registry.redhat.io/ubi8
```

Comment:
@AlyIbrahim should we switch over to `registry.access.redhat.com` since I had challenges accessing the image without additional permission changes. Travis testing will also fail as well

Reply:
@sabre1041 I changed to the suggested registry